styles.css

```diff
@@ -169,5 +169,5 @@ form {
   background:rgb(51, 158, 51);
   color: inherit;
   border-radius: 2px;
-  cursor: pointer;
+  cursor:crosshair;
```

Comment:
-great changes in CSS. However, I would like to suggest adding linters to your projects as A linter is great for identifying errors when you use standard rules, it would make your code automated by analyzing your source code to look for bugs and stylistic errors.